apps/web/src/app/[lang]/[username]/_components/profile-form.tsx

```diff
@@ -0,0 +1,139 @@
+'use client'
+
+import { Button } from '@/components/ui/button'
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@/components/ui/dialog'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form'
+import { Input } from '@/components/ui/input'
+import { useLanguage } from '@/context/language'
+import { changeProfileUsername } from '@/services/api/profiles'
+import { Profile } from '@/types/supabase'
+import { Dictionary } from '@/utils/dictionaries'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useRouter } from 'next/navigation'
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+
+import { z } from 'zod'
```

Comment:
For consistency, we can place imports in a more organized order,

Something like:
1. Lib/framewors imports (react, next, react-query, rhf)
2. Shared components
3. Shared components (if any)
4. Contexts
5. Utils/Services
6. Types